src/main/java/com/redislabs/modules/rejson/JReJSON.java

```diff
@@ -113,7 +113,7 @@ public JReJSON(Pool<Jedis> jedis) {
      * @param str the reply string to "analyze"
      * @throws RuntimeException
      */
-    private static void assertReplyNotError(final String str) {
+    protected static void assertReplyNotError(final String str) {
```

Comment:
We can trust a few private methods.

```suggestion
    private static void assertReplyNotError(final String str) {
```

Reply:
Making them private prevents me from testing them. I just make them package level instead.

Reply:
Skip testing this.

Reply:
Addressed, back to private